virtcontainers/qemu_arch_base_test.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	qemuArchBaseMachineType = "pc"
-	qemuArchBaseQemuPath    = "/usr/bin/qemu-lite-system-x86_64"
+	qemuArchBaseQemuPath    = "/usr/bin/qemu-system-x86_64"
```

Comment:
If we are dropping `qemu-lite` entirely, there are changes we'll need to make in a few other areas (search for `QemuPCLite`).

Reply:
I think the default will be using the vanilla 2.11 QEMU, though we are expecting a pc-lite supporting qemu-lite to be available in the future as well.  So, I think its reasonable to keep here for now?  WDYT @jodh-intel

Reply:
Ok - fine with this bit then ;)